### ISSUE TITLE
amDateFormat - momentConstructor$9 is not a function 

### ISSUE THREAD

**Issue by PillowPillow** [urish/ngx-moment#276]
**Description of the Issue and Steps to Reproduce:**

Did you search for duplicate issue? [Yes / No] **Yes**

Hello, i m facing an issue after upgrading from the v5.0.0 to the v6.0.1 (6.0.2 is broken cf https://github.com/urish/ngx-moment/issues/275). 
The pipe amDateFormat triggers an error "momentConstructor$9 is not a function".
```typescript
import * as moment from 'moment';
// ...
class MyComponent {
    // ...
    date = moment();
}
```
```html
<span>{{date|amDateFormat:'DD/MM/YYYY, HH:mm:ss'}}<span>
```

*Ensure your issue is isolated to ngx-moment. Issues involving third party tools will be closed unless submitted by the tool's author/maintainer.*
**Reproduced in an empty repository with only angular, moment@2.29.1 & ngx-moment@6.0.1**

**Environment:**

Please answer the following questions:

* Angular version? 
**13.1.1**
* TypeScript version?
**4.5.4**
* `moment` version?
**2.29.1**
* Are you using `moment-timezone`? 
**no**
* Are you using the angular-cli? 
**yes**
* Using Rollup/Webpack/System.js/Ionic/similar? 
**webpack through Angular Cli**

**Comment by pete-mcwilliams** [comment]
was fixed in 6.0.2 https://github.com/urish/ngx-moment/issues/245

**Comment by PillowPillow** [comment]
thank's @pete-mcwilliams .. so waiting for 6.0.3 with a fix for https://github.com/urish/ngx-moment/issues/275
